### ISSUE TITLE
Provider test packages no longer available

### ISSUE THREAD

**Issue by ErikEJ** [aspnet/EntityFramework#2889]
There now appear to be four dfiierent ASP.NET vNext galleries on MyGet:
1: https://www.myget.org/gallery/aspnetlitedev
2: https://www.myget.org/gallery/aspnetvnext

3: https://www.myget.org/gallery/aspnetliterelease
4: https://www.myget.org/gallery/aspnetrelease

I would like to update my product code with the most recent EF beta 7 build and run the test suite against the updated product code. This appears to be impossible at the moment, as current provider test packages do not seem to be available anywhere.

I assume that feed 1 is the one to be used going forward towards beta8 ? In that case, that feed is missing the three ".FunctionalTests " packages

I assume that feed 3 is the latest beta 7 bits, and that is where I should move to (build 15151 or later) in preparation for beta 7 - in that case, that feed is also missing the three ".FunctionalTests " packages.

I assume that feed feeds 2 and 4 will be decommisioned soon? Feed 2 seems to be in a bad state, with a mix in versions for EF product and test packages.

**Comment by bricelam** [comment]
@victorhurdugaci What do we need to do to get these published? We had [this code in our makefile.shade](https://github.com/aspnet/EntityFramework/blob/dev/makefile.shade#L11-L37) to enable it before.

**Comment by rowanmiller** [comment]
@ErikEJ just some general info on the dev feeds:
1 is the latest set of packages that all build together (unsigned and untested)
2 is the latest build from 1 that all the unit tests pass one and are signed

**Comment by ErikEJ** [comment]
@rowanmiller  Ah, now I get the meaning of "lite" - so 2 and 4 just need to be fixed regarding the missing packages

**Comment by ErikEJ** [comment]
@bricelam Any progress on this?

**Comment by bricelam** [comment]
I haven't dug into it yet. @victorhurdugaci any thoughts on why it doesn't "just work"?

**Comment by bricelam** [comment]
Dug into it. The new CI only runs the `compile` target, not `test-compile`. I'll send out a PR shortly.

**Comment by ErikEJ** [comment]
:star:

**Comment by victorhurdugaci** [comment]
Oh... You have test packages that have to be published? Can you move them to `src`?
Build v2 will do `pack` on `src` and `compile` on `test` and `samples`...

**Comment by bricelam** [comment]
No, they don't belong in `src`; they're tests. :wink: I'll update our sake script to copy them to `artifacts` during compile.

**Comment by ErikEJ** [comment]
@bricelam Will this fix also apply to the release feed at https://www.myget.org/gallery/aspnetrelease ? I think I need the packages in that feed in order to participate in the beta 7 release

**Comment by roji** [comment]
Am waiting on this as well

**Comment by rowanmiller** [comment]
Reopening to track getting this fixed for the release feed.

**Comment by bricelam** [comment]
I took a custom build of the packages locally and we're going to try and get them uploaded to the aspnetrelease feed. Please stand by...

**Comment by ErikEJ** [comment]
Standing by :pray: :smile:

**Comment by ErikEJ** [comment]
@bricelam Can you confirm that you expect no manual intervention to be required with beta 8, then feel free to close this :smile:

**Comment by bricelam** [comment]
Oh yeah, by the way, they should be uploaded now. :wink:

So long as KoreBuild doesn't change out from under us again, we should be good to go.

**Comment by roji** [comment]
FYI, I can see the beta7 release on nuget.org. However, I can't see a beta7 release for the two functional test suite nugets - AspNetRelease has beta7-15529.

This makes it impossible for my functional test suite project to reference beta7 (since it also needs to reference the test suites etc.)...

**Comment by ErikEJ** [comment]
@Roji I referenced AspNetRelease from all test and product projects, and passed the test suites. Then I pointed product library references to https://www.myget.org/gallery/aspnetbeta7take2/ (or now it could even be Nuget.org) and built those, and created the packages. But agree it would be ideal/easier if the https://www.myget.org/gallery/aspnetbeta7take2/ (or future type feed) also included the EF test packages.

**Comment by victorhurdugaci** [comment]
You shouldn't need to use `aspnetbeta7take2`. That was just a temporary feed for internal validation.

**Comment by roji** [comment]
Yeah, I can see the EntityFramework7.Core and Relational beta7 on nuget.org. However there's some sort of nuget dependency problem when trying to have EntityFramework7.Core-beta7 and EntityFramework7.Relational-beta7 alongside EntityFramework7.Core.FunctionalTests-beta7-15529 and EntityFramework7.Relational.FunctionalTests-beta7-15529. Haven't had time to investigate yet...

**Comment by ErikEJ** [comment]
@roji I think you misunderstood me - everything you need in order to run beta 7 verification test is available in http://myget.org/gallery/aspnetrelease - so reference v7.0.0-beta7-15540  in that feed!

**Comment by ErikEJ** [comment]
@victorhurdugaci In order to point to proper beta-7 packages and build against them before they are available on NuGet.org, I refer to that feed  aspnetbeta7take2  - which is no longer needed, agreed.

**Comment by roji** [comment]
@ErikEJ, I understand, and this is currently how I set things up. It seems very strange, however, that there is no EntityFramework7.Core.FunctionalTests and EntityFramework7.Relational.FunctionalTests anywhere that references beta7 (and not beta7-15540). This means that my functional tests project must reference beta7-15540 instead of beta7.

Not a showstopper but a bit problematic...

**Comment by ErikEJ** [comment]
@Roji Agree, but this is a work in Progress. I have logged https://github.com/aspnet/EntityFramework/issues/3016
